### PR TITLE
Setup Philippines Helpline

### PR DESCRIPTION
**Pull request by murilovmachado**
Primary reviewer: @stephenhand 

## Description
Sets up Philippines helpline.
Fixes some README files.

### Related Issues
Fixes [CHI-945](https://bugs.benetech.org/browse/CHI-945)

### Verification steps
After this is deployed, the Philippines helpline should be working.

### CODE REVIEW

File: twilio-iac/ecpat-staging/service-configuration-payload.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (stephenhand) [inline twilio-iac/ecpat-staging/service-configuration-payload.json:1]: This file should be unnecessary now we have the service configuration provisioner, it builds this config up from Terraform variables now.

**Review by stephenhand — APPROVED** [review]

File: twilio-iac/ecpat-staging/service-configuration-payload.json (hunk 1 above)
Comment (stephenhand) [reply twilio-iac/ecpat-staging/service-configuration-payload.json:1]: Sorry, the PR where these are removed still needs reviewing: https://github.com/techmatters/flex-plugins/pull/635

File: twilio-iac/README.md [hunk 2]
```diff
@@ -44,24 +44,25 @@ The process for a first run is as follows:
...
 8. _Optional:_ You can create a private `.tfvars` for the sensitive variables you can't check in values for - if you name it something ending in `private.tfvars` it will be ignored by git - or you can use TF_VAR_* environment variables for these as instructed above.
 
-9. Run the script below. Twilio creates a bunch of default resources on a new account and Aselo uses some of them. We need to import them into terraform first, otherwise terraform assumes they don't exist and will try to create them, resulting in errors.
+9. Run the script below from flex-plugins/scrips/ folder. Twilio creates a bunch of default resources on a new account and Aselo uses some of them. We need to import them into terraform first, otherwise terraform assumes they don't exist and will try to create them, resulting in errors.
 ```
-npm run twilioResources -- import-account-defaults <helpline>-<environment> [-v my-private.tfvars]]
+TWILIO_ACCOUNT_SID=XXX TWILIO_AUTH_TOKEN=xxx AWS_REGION=us-east-1 npm run twilioResources -- import-account-defaults <helpline>-<environment> [-v my-private.tfvars]]
```
[the remark this answers is not in the document]
Comment (murilovmachado) [reply twilio-iac/README.md:49]: Oh, I didn't know this way wouldn't work on Windows. I'll update the README to be more clear

[the remark this answers is not in the document]
Comment (GPaoloni) [reply twilio-iac/README.md]: Note of importance here: the script user in AWS does not have proper permissions to run this (leaves the Dynamo locks in a bad state). We should either specify to use personal credentials or to make the script user have the right set of permissions.

**Pull request merged**